build.gradle

```diff
@@ -100,7 +100,7 @@ def pluginList = [
 pluginBundle {
     website = 'https://github.com/progress/latte'
     vcsUrl = vcsBaseUrl
-    tags = ['abl', 'pct']
+    tags = ['abl', 'pct', 'openedge']
     description = "oe.espresso"
     version = "0.2.1"
```

Comment:
We also need to update this version to 0.2.2

Reply:
fixed it.